README.md

```diff
@@ -34,7 +34,7 @@ cd syne-tune
 pip install -e '.[extra]'
 ```
 
-You can see the FAQ [What are the different installations options supported?](docs/faq.md#installations) for more install options.
+We recommend you to install it in a virutal environment to avoid package inconsistencies. You can see the FAQ [What are the different installations options supported?](docs/faq.md#installations) for more install options.
```

Comment:
I'd say:
"When installing Syne Tune from sources, we recommend to use a virtual environment."

Reply:
BTW: Could you also change the code snippet above?
"pip install 'syne-tune[extra]==0.3.0'"
must become
"pip install 'syne-tune[extra]'"